docs/content/docs/concepts/users-accounts.mdx

```diff
@@ -380,6 +380,37 @@ export const auth = betterAuth({
 });
 ```
 
+#### Existing User Auto-Linking Policy
+
+The `existingUserMode` option controls when OAuth/SSO providers can auto-link to an existing user with the same email address. This provides fine-grained control over account linking security.
+
+```ts title="auth.ts"
+export const auth = betterAuth({
+    account: {
+        accountLinking: {
+            enabled: true,
+            existingUserMode: "trusted_providers_only", // recommended
```

Comment:
This really sounds like a policy to me. What do you think of renaming this option to `existingUserPolicy`?